clang-tools-extra/test/clang-tidy/checkers/daedalean-use-noexcept.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: %check_clang_tidy %s daedalean-use-noexcept %t
+
+class S {
+public:
+  S();
+  // CHECK-MESSAGES: :[[@LINE-1]]:3: warning: Function 'S' should be noexcept [daedalean-use-noexcept]
+  S(const S &)
+  noexcept = default;
+
+  bool operator==(const S &) const noexcept;
```

Comment:
add test for lambda

Reply:
Good point. Also separated in 3 matchers to provide better messages (and only match operator() in lambda).